index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Pacifico&family=Rubik+Moonrocks&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Project</title>
+</head>
+
+<body>
+    <header class="top-bar">
+        <a><h4>Welcome</h4> </a>
+      <nav>  <img class="menu" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAARMAAAC3CAMAAAAGjUrGAAAAMFBMVEX///8AAADl5eX09PTV1dUICAiurq5TU1Orq6tJSUnc3Nzt7e1ERET6+vpRUVG9vb0gCS9GAAAA8ElEQVR4nO3dOQ4CQRAEQZblPv//W2yUFsLoNSKekGb1SLPbAQAAAAAAAAAAAADfbodh9+kC9ViGXaYL1Hm6yWm6QGlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJnWdbrLBjXqd9pwuAAAAAMCvzstx1Ba3x9f09rjBJjbq0qQ0KU1Kk9KkNClNSpPSpDQpTUqT0qQ0KU1Kk9KkNClvQet9mnW5TRcAAAAAgP/d98PW6QJljy27fWlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJuWPs3LLAAAAAAAAAAAAAGAbPhzZL1DD5UvKAAAAAElFTkSuQmCC"
+            alt="Menu Icon"> </nav>
+    </header>
+    <main>
+      <section> 
+        <div class="space"></div>
+              <h1 class="header">Hey there. I'm Cisco.</h1>
+
+               <h1 class="article">I'm a software developer.</h1>
+
+        <div>
+            <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need developed, don't hesitate to contact me.
+            </p>
+        </div>
+        <section><div class="space"></div></section>
+        <div class="images">
+            <ul>
+           <li> <a href="#"><img src="Images/twittertwitter.png" alt="Twitter logo"></a></li>
+            <li><a href="#"><img src="Images/Linkedin.png" alt="LinkedIn logo"></a></li>
+            <li><a href="#"><img src="Images/mediummedium.png" alt=" Medium logo"></a></li>
+            <li><a href="#"><img src="Images/githubgithub.png" alt="Github logo"></a></li>
+           <li><a href="#"><img src="Images/angellistangelist.png" alt="Angellist logo"></a></li>
+            </ul>
```

Comment:
- [Optional] **Recommended** Please align the icons for social media centered as it is on the Figma design. 

##### YOur design
![image](https://user-images.githubusercontent.com/58771507/186895063-35a4fd3c-28b9-430c-9666-baed1e08d84c.png)

##### Figma design
![image](https://user-images.githubusercontent.com/58771507/186895150-b8b425d1-375d-4af7-b682-36f501554289.png)